CHANGELOG.md

```diff
@@ -3,9 +3,95 @@
 
+## 4.0.0
+
+_Aug 24, 2021_
+
+🎉 This is the first official stable release of the data grid component 🎉!
+
+We have been iterating on the component for [18 months](https://github.com/mui-org/material-ui-x/commit/705cb0f387b5f3aa056bf40c4183a2342b317447). With the latest introduction of the [row edit](https://material-ui.com/components/data-grid/editing/#row-editing) feature, a load of bug fixes, and a polish of the documentation. We believe the component deserves a stable release.
+
+The MUI X v4.0.0 release supports [MUI Core](https://github.com/mui-org/material-ui) v4 and has partial support for v5-beta. With the soon-to-be-released v5 version of the core components, we are moving ongoing work to the v5 release line (Core and X).
+The support for existing developers on MUI v4 might be degraded going forward, we encourage you to migrate to MUI Core v5-beta and soon MUI X v5-beta.
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce the [row editing](https://material-ui.com/components/data-grid/editing/#row-editing) feature (#2098) @m4theushw
+
+  <img src="https://user-images.githubusercontent.com/3165635/130665023-3c0730ab-502e-4da1-8bc1-d572427ad2d6.gif" width="851" height="382" />
+
+- ⚡️ Rename the name of the `XGrid` component to `DataGridPro` (#2382) @m4theushw
+
+  This should help clarify the products vs. plans separation. [MUI X](https://github.com/mui-org/material-ui-x) is a product line on its own. It contains MIT and Commercial software. Removing X from the name of the paid components should help remove a possible confusion: the MIT version of X is meant to be valuable enough for developers to use it, without feeling that it's crippled compared to other OSS alternatives.
+  The Pro prefix should help make it clear what's MIT and what's not.
+- ✨ Rename the `@material-ui` npm scope to `@mui` (#2341) @oliviertassinari
+
+  This is part of the ongoing rebranding of the project and company. Material-UI is our current official name. However, we are going to change it. It's too long to write, read, pronounce. It associates us top much with Material Design. In the near future, the whole project/company is moving to MUI and https://mui.com/.
+- 💡 The `api` property was removed from the callback params. To access the API, use the `DataGridPro` (#2312) @DanailH
+
+### `@mui/x-data-grid@v4.0.0` / `@mui/x-data-grid-pro@v4.0.0`
+
+#### Breaking changes
+
+- [DataGrid] Move packages to `@mui` scope and rename `XGrid` to `DataGridPro` (#2341, #2382) @m4theushw @oliviertassinari
+  You can find in the above highlight section why we are making these name changes. You can migrate following these steps:
+
+  ```diff
+  -import { DataGrid } from '@material-ui/data-grid';
+  +import { DataGrid } from '@mui/x-data-grid';
+  ```
+
+  ```diff
+  -import { XGrid } from '@material-ui/x-grid';
+  +import { DataGridPro } from '@mui/x-data-grid-pro';
```

Comment:
```suggestion
  +import { DataGridPro as XGrid } from '@mui/x-data-grid-pro';
```

Reply:
This seems like something devs can add themselves. I think it's better to just move away from `XGrid`.

Reply:
The diff is incomplete without either this, or an example of renaming a call. This is a working diff, and makes it clear that there's more to do to clean up the naming.